python/lsst/daf/butler/script/exportCalibs.py

```diff
@@ -170,7 +171,7 @@ def exportCalibs(
         log.info("Saving %d dataset(s)", len(datasetsToExport))
         export.saveDatasets(datasetsToExport)
 
-    sortedDatasets = sorted(datasetsToExport, key=lambda x: x.datasetType.name)
+    sortedDatasets = sorted(datasetsToExport, key=attrgetter("datasetType.name", "dataId"))
```

Comment:
:exploding_head: I had no idea `attrgetter` handled dot-separated names.

Reply:
Neither did I until I read the documentation...